softwareconfiguration/mlplan/src/test/java/de/upb/crc901/mlplan/multiclass/wekamlplan/sklearn/SKLearnClassifierFactoryTest.java

```diff
@@ -0,0 +1,60 @@
+package de.upb.crc901.mlplan.multiclass.wekamlplan.sklearn;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import hasco.model.ComponentInstance;
+import jaicore.ml.WekaUtil;
+import jaicore.ml.core.evaluation.measure.singlelabel.ZeroOneLoss;
+import jaicore.ml.scikitwrapper.ScikitLearnWrapper;
+import weka.core.Instances;
+
+public class SKLearnClassifierFactoryTest {
+
+	private static final File TEST_PIPE = new File("testrsc/sklearn/sklearn_classifier_factory.pipe_description.json");
+	private static ComponentInstance componentInstance;
+	private static SKLearnClassifierFactory factory;
+
+	@BeforeClass
+	public static void setup() throws IOException {
+		// ObjectMapper om = new ObjectMapper();
+		// componentInstance = ComponentInstance.fromJson(new ObjectMapper().readTree(FileUtil.readFileAsString(TEST_PIPE)));

```

Comment:
commented code?

Reply:
resolved, will be completed in another branch anyway.